Package.swift

```diff
@@ -25,6 +25,7 @@ let package = Package(
             ],
             path: "Auth0",
             exclude: ["Info.plist"],
+            resources: [.copy(./Resources/PrivacyInfo.xcprivacy)]
```

Comment:
The syntax is invalid and does not compile. Did you test the changes on a sample app that uses the Swift Package Manager?

<img width="803" alt="Screenshot 2024-04-12 at 13 54 37" src="https://github.com/auth0/Auth0.swift/assets/5055789/6f273b53-0846-4755-a92a-cf76f61171c0">